services/filemanager/azureBlobStoragemanager.go

```diff
@@ -105,10 +123,26 @@ func (manager *AzureBlobStorageManager) Upload(ctx context.Context, file *os.Fil
 		return UploadOutput{}, err
 	}
 
-	return UploadOutput{Location: blobURL.String(), ObjectName: fileName}, nil
+	return UploadOutput{Location: manager.blobLocation(&blobURL), ObjectName: fileName}, nil
 }
 
-func (manager *AzureBlobStorageManager) ListFilesWithPrefix(ctx context.Context, startAfter, prefix string, maxItems int64) (fileObjects []*FileObject, err error) {
+func (manager *AzureBlobStorageManager) shouldCreateContainer() bool {
```

Comment:
minor and subjective
```suggestion
func (manager *AzureBlobStorageManager) createContainer() bool {
```

Reply:
Will make the changes.

Reply:
Done.